apps/crn-server/src/controllers/team.controller.ts

```diff
@@ -60,6 +60,7 @@ export default class TeamController {
       throw new NotFoundError(undefined, `team with id ${teamId} not found`);
     }
 
+    console.log(team);
```

Comment:
forgotten dev log

Reply:
I believe you were seeing an old version